README.md

```diff
@@ -1,22 +1,141 @@
-[![Build Status](https://travis-ci.org/gcovr/gcovr.png?branch=master)](https://travis-ci.org/gcovr/gcovr) 
+gcovr
+=====
 
-Gcovr Overview
---------
+generate GCC code coverage reports
+
+[website] • [documentation] • [bugtracker] • [GitHub][repo] • [![Build Status][travis-ci-badge]][travis-ci]
```

Comment:
maybe a pipy badge could be a good thing too ?
[![PyPi](https://img.shields.io/pypi/v/gcovr.svg)](https://pypi.python.org/pypi/gcovr)